uber/models.py

```diff
@@ -720,14 +720,19 @@ def affiliates(self):
 
     @classmethod
     def model_mixin(cls, model):
```

Comment:
As you can see, the fix is to simply check for ``SessionMixin`` and ``QuerySubclass`` as special cases, and fall back to checking models.